indico/modules/events/abstracts/templates/emails/comment.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'events/abstracts/emails/base.html' %}
+
+{% block subject_message %}New abstract comment{% endblock %}
+
+{% block header_subtitle %}New abstract comment{% endblock %}
+
+{% block header_subtitle_ref %}ref #{{ abstract.friendly_id }}{% endblock %}
+
+{% block content_header %}
+    <p>Dear {{ recipient.first_name }},</p>
+    <p>{{ submitter.full_name }} commented on the abstract <strong>{{ abstract.title }}</strong>:</p>
+    <blockquote style="border-left: 5px solid #dfdfdf; padding-left: 10px; font-style: italic; color: #999;">
+        {{ comment }}
+    </blockquote>
+{% endblock %}
+
+{% block content_body %}
+    <p>You can view this comment on the abstract page:</p>
+    <p>
+        <a href="{{ url_for('abstracts.display_abstract', abstract, _external=true) }}">
```

Comment:
This link isn't correct for contributors/reviewers/conveners as they do not have management area access